Nos/Views/Home/HomeFeedView.swift

```diff
@@ -11,7 +11,9 @@ struct HomeFeedView: View {
     @Environment(CurrentUser.self) var currentUser
     @ObservationIgnored @Dependency(\.analytics) private var analytics
 
-    @State private var refreshController: RefreshController = DefaultRefreshController()
+    @State private var refreshController: RefreshController = DefaultRefreshController(
+        lastRefreshDate: Date(timeIntervalSince1970: Date.now.timeIntervalSince1970 + Double(Self.staticLoadTime))
```

Comment:
TimeInterval is a type alias for Double, so there is no need to cast it. Also, it would be more succinct to just do:
`lastRefreshDate: Date.now + Self.staticLoadTime`

Reply:
I think I just copied what used to be here but you're right. I'll fix it.